Assets/Plugins/WebSocket/WebSocket.cs

```diff
@@ -429,7 +429,7 @@ public class WebSocket : IWebSocket {
             }
           }
         }
-      } catch (Exception e) {
+      } catch (Exception) {
```

Comment:
IMO: implement error handling which uses the Exception, then just removing it.

Reply:
What did you have in mind for error handling? IMO it doesn't look like there's any reasonable error handling to be done. The only thing we'd use the exception for would be to log its message and stack trace, but this plugin doesn't write any logs.